ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -666,7 +784,7 @@ export const RuntimePanel = fp.flow(
   const initialPanelContent = fp.cond([
     // currentRuntime being undefined means the first `getRuntime` has still not completed.
     [([r, ]) => r === undefined, () => PanelContent.Customize],
-    [([r, s]) => r === null || s === RuntimeStatus.Unknown, () => PanelContent.Create],
+    [([r, s]) => r === null || s === RuntimeStatus.Unknown || s === RuntimeStatus.Deleted, () => PanelContent.Create],
```

Comment:
The previous logic was correct. It will need to be updated once https://precisionmedicineinitiative.atlassian.net/browse/RW-5922 is fixed, but it would be looking at the labels instead in that case. As it currently stands, you'll only see Deleted on the client side if the deleted runtime was a user override (in which case, we want to show the customize page instead).

Reply:
ah, this would be the source of the behavior you asked me about in Slack, then. I think I did this after talking with Lou about the delete flow.